src/main/java/org/folio/readingroom/controller/PatronPermissionController.java

```diff
@@ -16,12 +16,19 @@
 public class PatronPermissionController implements org.folio.readingroom.rest.resource.ReadingRoomPatronPermissionApi {
   private final PatronPermissionsService patronPermissionsService;
 
+  @Override
+  public ResponseEntity<List<PatronPermission>> getPatronPermissionsByUserId(UUID patronId, UUID servicePointId) {
+    log.info("getPatronPermissionsByUserId:: Updating patronPermission for patron {}", patronId);
+    return ResponseEntity.status(HttpStatus.OK)
+      .body(patronPermissionsService.getPatronPermissionsByUserId(patronId, servicePointId));
+  }
+
   @Override
   public ResponseEntity<List<PatronPermission>> updatePatronReadingRoomPermission(UUID patronId,
-    List<PatronPermission> patronPermission) {
+    List<PatronPermission> patronPermission, UUID servicePointId) {
```

Comment:
I think servicepointId is not needed for this API

Reply:
fixed